docs/application-connector/05-05-application-connectivity-certs-setup-job.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Application Connectivity Certs Setup Job
+type: Configuration
+---
+
+To configure the Application Connectivity Certs Setup Job, override the default values of its `values.yaml` file. This document describes parameters that you can configure.
+
+>**TIP:** To learn more about how to use overrides in Kyma, see the following documents: 
+>* [Helm overrides for Kyma installation](/root/kyma/#configuration-helm-overrides-for-kyma-installation)
+>* [Sub-charts overrides](/root/kyma/#configuration-helm-overrides-for-kyma-installation-sub-chart-overrides)
+
+
+## Configurable parameters
+
+This table lists the configurable parameters, their descriptions, and default values:
+
+| Parameter | Description | Default value |
+|-----------|-------------|---------------|
+| **global.applicationConnectorCaKey** | Specifies the base64 encoded private key for the Application Connector. If not provided it will be generated. | `` |
```

Comment:
```suggestion
| **global.applicationConnectorCaKey** | Specifies the base64-encoded private key for the Application Connector. If not provided it will be generated. | `` |
```